Prox/Prox/AppDelegate.swift

```diff
@@ -38,6 +39,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func applicationWillEnterForeground(_ application: UIApplication) {
         // Called as part of the transition from the background to the active state; here you can undo many of the changes made on entering the background.
+        placeCarouselViewController!.refreshLocation()
```

Comment:
You will also need to add this line to `applicationDidBecomeActive`. When the app starts from cold, ` applicationDidBecomeActive` is called. When it is awoken suspension, then `applicationWillEnterForeground` is called. If you do not request user location in `applicationDidBecomeActive` then we will never get a request to monitor location from a new install.